db/migrate/20200227105316_add_user_notifications_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddUserNotificationsTable < ActiveRecord::Migration[6.0]
+  def change
+    create_table :user_notification do |t|
+      t.integer :user_id, null: false
+      t.string :provider_code, null: false
```

Comment:
maybe index  something similar to 

https://github.com/DFE-Digital/teacher-training-api/blob/f4cedf7c693592147600f4131405b21be343eb9d/db/migrate/20190711074433_change_course_accrediting_provider_id_to_code.rb#L8

Reply:
:eyes: